src/pde_solve_ns.jl

```diff
@@ -102,5 +111,75 @@ function DiffEqBase.solve(
 
     Flux.train!(loss_n_sde, ps, data, opt; cb = cb)
 
-    save_everystep ? iters : re1(p3)(X0)[1]
+
+    if give_limit == false
+        save_everystep ? iters : re1(p3)(X0)[1]
+    else
+    ## UPPER LIMIT
+        sdeProb = SDEProblem(μ , σ , x0 , tspan , noise_rate_prototype = zeros(Float32,d,d))
+        ensembleprob = EnsembleProblem(sdeProb)
+        sim = solve(ensembleprob, EM(), EnsembleThreads(), dt=dt, trajectories=trajectories_upper,kwargs...)
+        ts = tspan[1]:dt:tspan[2]
+        function sol_high()
```

Comment:
don't write this out: instead write this as an SDEProblem ensemble and solve it with the algorithm the user gives.